src/dune_rules/module.mli

```diff
@@ -61,7 +61,7 @@ val path : t -> Module_name.Path.t
 
 val source : t -> ml_kind:Ml_kind.t -> File.t option
 
-val pp_flags : t -> (string list Action_builder.t * Sandbox_config.t) option
+val pp_flags : t -> (Command.Args.any Command.Args.t * Sandbox_config.t) option
```

Comment:
The old type was actually superior because it prevented the flags from introducing hidden targets. Command.Args.t doesn't do that. But even without that, the `string list Action_builder.t` is strictly less powerful and enough for this job. Why go for the bigger hammer?